src/Board.hpp

```diff
@@ -12,25 +12,25 @@ namespace nBoard
 }
 class Board
 {
-public:
-    Board();
-    ~Board();
+    public:
+        Board();
+        ~Board();
 
-    const Field &getField( const int x, const int y) const;
+        const Field &getField( const int x, const int y) const;
 
-    bool isFildAlreadyVisited(const std::pair<int, int> &stepCoordinate);
-    bool isAllFieldsIsVisited();
+        bool isFildAlreadyVisited(const std::pair<int, int>& stepCoordinate);
+        bool isAllFieldsIsVisited();
```

Comment:
You must make choose what kind of variant will be final: "AlreadyVisited" or "IsVisited". Must be uniformity.
Thanks.